update_info.py

```diff
@@ -49,9 +55,22 @@ def get_description_url(element):
 
 for key, info in entryInfo.items():
     if key not in originalEntryInfo:
+        for k,v in info.items():
+            print(k,v)
+        available = valid_input('Make this addon available? (y/n) ')
+        info['available'] = True if available == 'y' else False
```

Comment:
Seems like this could be simplified since its just a `y/n`. 

```py
info['available'] = True if input('Make this addon available? (y/n) ') == 'y' else False
```

Doesn't really matter I suppose, and this is probably a nitpick.

Reply:
Might as well tidy it up a little. Since the `valid_input` function I made wraps things in a while loop that checks for invalid input, you can accidentally hit "t" without having to start over. Thanks!